profiles/common.go

```diff
@@ -61,6 +61,23 @@ func ParseMap(s string) (map[string]string, error) {
 	return params, nil
 }
 
+func ParseList(s string)([]string, error) {
```

Comment:
I guess this should be in utils.go

Reply:
This file also implements `ParseMap` which is also used for parsing the input profiles in the CLI so I've kept it here for consistency.